UCSD/campus-mobile
# Offline mode

## Summary
[Network Connectivity: Offline mode#1664](https://app.zenhub.com/workspaces/campus-mobile-replatform-59791ae0251e9315c45be418/issues/ucsd/campus-mobile/1664)

## Changelog

[General] [Add] - Add an alert dialog when detecting the app is offline


## Test Plan

1. Open the app and go to a page, then turn off the network connection. The correct behavior should be an alert dialog pop up about offline network status.
2. Turn off the network connection first, then open the app and go to a page. The offline alert dialog should appear right when the home page is shown.
3. The above testing process should be performed on every page.
[Demo video](https://drive.google.com/file/d/1KGwug0_8z8R-arRQnJ10XoTgP-93PN2T/view?usp=sharing) here
